scripts/setup-minikube-for-linux.sh

```diff
@@ -19,7 +19,7 @@ touch $HOME/.kube/config
 export KUBECONFIG=$HOME/.kube/config
 sudo -E minikube start --vm-driver=none
 sudo -E minikube addons enable ingress
-sudo -E chmod a+r ~/.minikube/client.key
+#sudo -E chmod a+r ~/.minikube/client.key
```

Comment:
why?

Reply:
apparently it's not longer there, and no longer needed...